tardis/io/model_reader.py

```diff
@@ -85,19 +85,71 @@ def read_abundances_file(abundance_filename, abundance_filetype,
     """
 
     file_parsers = {'simple_ascii': read_simple_ascii_abundances,
-                    'artis': read_simple_ascii_abundances}
+                    'artis': read_simple_ascii_abundances,
+                    'tardis_model': read_simple_isotope_abundances}
+
+    isotope_abundance = pd.DataFrame()
+    if abundance_filetype == 'tardis_model':
+        index, abundances, isotope_abundance = read_simple_isotope_abundances(
+            abundance_filename)
+    else:
+        index, abundances = file_parsers[abundance_filetype](
+            abundance_filename)
```

Comment:
If, `read_simple_ascii_abundances` function returns an extra `None` (instead of `(index,abundance)`, returning `index,abundance,None`)) , then we can remove this. But I don`t think , its a good idea.